NEWS

```diff
@@ -1,3 +1,6 @@
+42.3.20170622
```

Comment:
My bad, I will remove the changes in the updated PR. But then, add this to the readme so next contributor will not get confused ;-)